preprocessor/folder/file.go

```diff
@@ -106,16 +142,9 @@ func handleNewFile(pgPool *pgxpool.Pool, filePath string) (string, error) {
 
 // shouldIncludeFile reports whether the file should be included in the processing
 func shouldIncludeFile(fileName string) bool {
-	whiteListExtension := config.GetConfig().FolderPreprocessor.FileExtensions
 	if fileName[0] == '.' {
 		return false
 	}
 	fileExt := filepath.Ext(fileName)
-	if fileExt == ".partial" {
-		return false
-	}
-	if slices.Contains(whiteListExtension, fileExt) {
-		return true
-	}
-	return false
+	return fileExt == ".partial"
```

Comment:
```suggestion
	return fileExt != ".partial"
```

As I understand it, this function should return `false` when the file extension is `.partial`.

Reply:
yes dumb error when fixing linter warning, thanks for catching